hive-event-listeners/apiary-gluesync-listener/src/main/java/com/expediagroup/apiary/extensions/gluesync/listener/ApiaryGlueSync.java

```diff
@@ -76,8 +82,45 @@ public ApiaryGlueSync(Configuration config, AWSGlue glueClient, String gluePrefi
   }
 
   @Override
-  public void onCreateTable(CreateTableEvent event) throws MetaException {
-    if (event.getStatus() == false) {
+  public void onCreateDatabase(CreateDatabaseEvent event) {
+    if (!event.getStatus()) {
+      return;
+    }
+    Database database = event.getDatabase();
+    try {
+      CreateDatabaseRequest createDatabaseRequest = new CreateDatabaseRequest()
```

Comment:
is it possible to add properties to the database so we can mark it as being created by apiary-glue-sync process?
Similarly if we drop it we can check for this property to have a safety we don't drop databases non synced by us?

Reply:
i've added it in the params

Reply:
Actually we first need to get the glue db object in a separate request to get its params cause this event only gives us the hive db info. I updated the logic for that